src/cli/cli_tcat.cpp

```diff
@@ -279,6 +321,15 @@ template <> otError Tcat::Process<Cmd("start")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli tcat stop
+ * @code
+ * tcat stop
+ * Done
+ * @endcode
+ * Stops TCAT operation.
```

Comment:
Need a @par before this line so the command description is pulled into our CLI reference

Reply:
I'm not sure if I got the line correctly please double check.